main.go

```diff
@@ -99,6 +99,11 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+	if len(os.Args) > 1 {
+		// Use the CLI to run a task other than starting the server.
+		runTask(os.Args[1], db)
+		os.Exit(0)
+	}
```

Comment:
Is the idea that we'd run two separate daemons-- the regular `backend` and this task? If so, I think I'd prefer if we created a separate package for each executable since the modes of operation are so different!